sdk/monitor/opentelemetry-exporter/test/common/assert.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as assert from "assert";
+import { Base, Envelope } from "../../src/Declarations/Contracts";
+import {
+  AI_OPERATION_ID,
+  AI_OPERATION_PARENT_ID
+} from "../../src/utils/constants/applicationinsights";
+import { Expectation } from "./scenario/types";
+
+export const assertData = (actual: Base, expected: Base): void => {
```

Comment:
why `export const foo = function() {}` instead of `export function foo() {}`?

Reply:
I try to use arrow functions when `this` isn't necessary. Is using `function` preferable anyways?

Reply:
This is just a style thing. There is no functional difference between them, I just thought it was interesting. :)